tests/functional/restart/38-auto-restart-stopping.t

```diff
@@ -52,7 +52,7 @@ ${BASE_GLOBAL_CONFIG}
 "
 
 run_ok "${TEST_NAME}-workflow-start" cylc play "${WORKFLOW_NAME}" --host=localhost
-cylc workflow-state "${WORKFLOW_NAME}//1/foo:running" --interval=1 --max-polls=20 >& $ERR
+cylc workflow-state "${WORKFLOW_NAME}//1/foo:started" --messages --interval=1 --max-polls=20
```

Comment:
The redirection to `$ERR` was deliberately added by @oliver-sanders by the look of it - probably there to capture or hide command output depending on debug mode. Does it need to be removed?

Reply:
It was hiding the failure of this command to do anything useful - without `--messages`, the `:started` modifier doesn't do anything (except return a warning that started is not a task state)